### PR TITLE
PL-1093 + PL-1096: Teida Integration

### PR DESCRIPTION
**Pull request by usman-farooq-bboxx**
change log:
1- Added Jackal pcb creation documentation
2- Added Home3 product creation documentation
3- Added both end points reference

### CODE REVIEW

**Review by mujeeb-bboxx — CHANGES_REQUESTED** [review]
Do not use real values (e.g. imei, device key etc.). I would suggest amending this commit (so that real values are no longer visible).
Refer to the documentation of other endpoints  and use values that are being used there.
e.g. https://github.com/BBOXX/slate/blob/master/source/includes/_create_tiger_pcb.md
Use "010101234560005" as `imei` and "4ab1ab3ab1ab7ab9ab1ab4ab" as `device_key`.

**Review by mujeeb-bboxx — CHANGES_REQUESTED** [review]
`imei` and `device_key` were just **some** examples, I can still see real values for other fields. Please update those too.
Also, please ensure that this PR only contains one commit.

**Review by barkatkhan-bboxx — APPROVED** [review]

**Comment by usman-farooq-bboxx** [comment]
> `imei` and `device_key` were just **some** examples, I can still see real values for other fields. Please update those too.
> Also, please ensure that this PR only contains one commit.

The PR can not be done in one commit (will do it later on), also as requested the data has been changed accordingly.

**Review by z-adeel-bboxx — APPROVED** [review]

File: source/includes/_create_jackal_pcb.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+         "data": {"pcb": {"part_type_id": 44,
+                          "properties": {"device_key": "4ab1ab3ab1ab7ab9ab1ab4ab",
+                                         "imei": "010101234560005",
+                                         "imsi": "213514135171353",
+                                         "serial_number": "HB000501-084032-G"},
+                          "serial_number": "010101234560005",
+                          "manufacturer": "A Factory"},
+                  "sim": {"part_type_id": 43,
+                          "serial_number": "213514135171353"}},
+     }
```
Comment (mujeeb-bboxx) [inline source/includes/_create_jackal_pcb.md]: The actual response is different. The documentation is meant to reflect the actual request/response, please update the documentation accordingly.

**Review by mujeeb-bboxx — CHANGES_REQUESTED** [review]

File: source/includes/_create_product_home3.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+    r = requests.post(url=url, data=data, headers=headers)
+
+    print r.json()
+    >>> {
+        'data': {
+            'battery': {
+                'part_id': 18024, 
+                'part_type_id': 18004, 
+                'serial_number': 'madeupserialnumb3r'
```
Comment (mujeeb-bboxx) [inline source/includes/_create_product_home3.md]: This serial_number is made up but others seem real. Please make them consistent.

File: source/includes/_create_product_home3.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+                'part_id': 18023, 
+                'part_type_id': 158, 
+                'properties': {
+                    'device_key': '000000000000000000000004', 
+                    'erp_code_override': 'PA04010102', 
+                    'iccid': '66685607592181745482', 
+                    'imei': '110932356179789', 
+                    'imsi': '748655857675019', 
+                    'serial_number': 'PA040000-018002-9'
+                    },
```
Comment (mujeeb-bboxx) [inline source/includes/_create_product_home3.md]: Fix indentation.

File: source/includes/_create_product_home3.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+                'properties': {
+                    'device_key': '000000000000000000000004', 
+                    'erp_code_override': 'PA04010102', 
+                    'iccid': '66685607592181745482', 
+                    'imei': '110932356179789', 
+                    'imsi': '748655857675019', 
+                    'serial_number': 'PA040000-018002-9'
+                    }, 
+                'serial_number': '110932356179789'
+                },
```
Comment (mujeeb-bboxx) [inline source/includes/_create_product_home3.md]: Fix indentation.

File: source/includes/_create_product_home3.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+                }, 
+            'product': {
+                'device_key': '000000000000000000000004', 
+                'iccid': '66685607592181745482', 
+                'imei': '110932356179789', 
+                'imsi': '748655857675019', 
+                'product_imei': '110932356179789', 
+                'product_type_id': 10, 
+                'serial_number': 'PA040000-018002-9'
+                },
```
Comment (mujeeb-bboxx) [inline source/includes/_create_product_home3.md]: Fix indentation.

File: source/includes/_create_product_home3.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+                'product_imei': '110932356179789', 
+                'product_type_id': 10, 
+                'serial_number': 'PA040000-018002-9'
+                }, 
+            'sim': {
+                'part_id': 18022, 
+                'part_type_id': 43, 
+                'serial_number': '66685607592181745482'
+                }
+                },
```
Comment (mujeeb-bboxx) [inline source/includes/_create_product_home3.md]: Fix indentation.

File: source/includes/_create_product_home3.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+
+```python
+    url = "https://smartapi.bboxx.co.uk/v1/products/create_product/home3"
+    data = json.dumps({
+        'pcb_imei': '110932356179789',
+        'battery': {
+            'part_type_id': 18004, 
+            'serial_number': 'madeupserialnumb3r'
+            }
+            })
```
Comment (mujeeb-bboxx) [inline source/includes/_create_product_home3.md]: Fix indentation.

**Review by mujeeb-bboxx — CHANGES_REQUESTED** [review]

File: source/includes/_create_product_home3.md (hunk 3 above)
Comment (usman-farooq-bboxx) [reply source/includes/_create_product_home3.md]: done

File: source/includes/_create_product_home3.md (hunk 4 above)
Comment (usman-farooq-bboxx) [reply source/includes/_create_product_home3.md]: done

File: source/includes/_create_product_home3.md (hunk 2 above)
Comment (usman-farooq-bboxx) [reply source/includes/_create_product_home3.md]: done

File: source/includes/_create_product_home3.md (hunk 7 above)
Comment (usman-farooq-bboxx) [reply source/includes/_create_product_home3.md]: done

Comment (mujeeb-bboxx) [reply source/includes/_create_product_home3.md]: Others have been fixed, but this one still needs to be fixed.

**Review by mujeeb-bboxx — CHANGES_REQUESTED** [review]

**Review by mujeeb-bboxx — APPROVED** [review]

**Pull request merged**